app/tests/unit/mutations/application/createApplicationMutation.test.ts

```diff
@@ -12,7 +12,6 @@ const mutation = `
     $input: CreateApplicationMutationChainInput!
   ) {
     createApplicationMutationChain(input: $input) {
-      clientMutationId
```

Comment:
Is clientMutationId no longer needed for relay's optimism? It's just an echoback field: it's a no-op server-side.

Reply:
It's not needed since https://github.com/facebook/relay/releases/tag/v8.0.0